integration_test/integration_test.go

```diff
@@ -115,23 +118,28 @@ var _ = Describe("DatadogFirehoseNozzle", func() {
 			Expect(metric.Type).To(Equal("gauge"))
 
 			if metric.Metric == "cloudfoundry.nozzle.origin.metricName" || metric.Metric == "cloudfoundry.nozzle.metricName" {
-				Expect(metric.Tags).To(HaveLen(4))
-				Expect(metric.Tags[0]).To(Equal("deployment:deployment-name"))
-				if metric.Tags[1] == "job:doppler" {
+				Expect(metric.Tags).To(HaveLen(9))
+				Expect(metric.Tags[0]).To(Equal("deployment:deployment-name-aaaaaaaaaaaaaaaaaaaa"))
+				if metric.Tags[4] == "job:doppler" {
 					Expect(metric.Points).To(Equal([]metrics.Point{
 						{Timestamp: 1, Value: 5.0},
 					}))
-				} else if metric.Tags[1] == "job:gorouter" {
+				} else if metric.Tags[4] == "job:gorouter" {
 					Expect(metric.Points).To(Equal([]metrics.Point{
 						{Timestamp: 2, Value: 10.0},
 					}))
 				} else {
 					panic("Unknown tag")
 				}
 			} else if metric.Metric == "cloudfoundry.nozzle.origin.counterName" || metric.Metric == "cloudfoundry.nozzle.counterName" {
-				Expect(metric.Tags).To(HaveLen(4))
-				Expect(metric.Tags[0]).To(Equal("deployment:deployment-name"))
-				Expect(metric.Tags[1]).To(Equal("job:doppler"))
+				Expect(metric.Tags).To(HaveLen(7))
+				Expect(metric.Tags[0]).To(Equal("deployment:deployment-name-aaaaaaaaaaaaaaaaaaaa"))
+				Expect(metric.Tags[1]).To(Equal("deployment:deployment-name_env_name"))
```

Comment:
can we also add the tag `deployment:deployment-name`?

Reply:
done

Reply:
thanks!